_redirects

```diff
@@ -1,2 +1,5 @@
-/api/* https://api.fr.openfisca.org/api/:splat 200
-/legislation/* https://api.fr.openfisca.org/legislation/:splat 200
\ No newline at end of file
+/api/v21/* https://v21.api.fr.openfisca.org/:splat 200
```

Comment:
@Morendil Are we keeping the `https://fr.openfisca.org/api/v21` url for users? 🤔

Reply:
Je crois qu'on peut parler français sur le dépôt français du site :)

Le plus simple pour moi c'était de reproduire l'existant puisque le focus de l'action était de changer de mode d'hébergement, pas de mettre en oeuvre une décision de publication (qu'on a d'ailleurs apparemment des difficultés à réaliser alors même que la décision est prise).

Maintenant qu'on a la possibilité de le faire avec ces redirections, on pourrait aussi rediriger v21 vers v24, et publier une nouvelle version "stable". Mais je ferais bien ça dans une PR séparée, pour ne pas confondre les deux actions.